<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="grass_robocup">
    <link name="link">
      <pose>0 0 0 0 0 0</pose>
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://grass_robocup/meshes/model.dae</uri>
          </mesh>
        </geometry>
      </visual>
    </link>
    <static>1</static>
  </model>
</sdf>
